diagnostic_rt_qpcr/operation_types/prepare_rt_qpcr_plate/protocol.rb

```diff
@@ -1,112 +1,188 @@
+# typed: false
 # frozen_string_literal: true
 
-needs 'Diagnostic RT-qPCR/PrepareRTqPCRValidation'
-needs 'Diagnostic RT-qPCR/PrepareqPCRPlateHelper'
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRHelper'
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRDebug'
+needs 'Diagnostic RT-qPCR/DataAssociationKeys'
+needs 'PCR Libs/PCRComposition'
+needs 'Microtiter Plates/MicrotiterPlate'
+needs 'Collection Management/CollectionTransfer'
 
 # Protocol for setting up a plate with extracted RNA samples
 #
 # @author Devin Strickland <strcklnd@uw.edu>
 # @author Cannon Mallory <malloc3@uw.edu>
 class Protocol
-  include PrepareqPCRPlateHelper
-  include PrepareRTqPCRValidation
+  include DiagnosticRTqPCRHelper
+  include DiagnosticRTqPCRDebug
+  include MicrotiterPlates
+  include CollectionTransfer
+  include DataAssociationKeys
 
+  METHOD = :sample_layout
```

Comment:
Why change from `:cdc_sample_layout`?

Reply:
Oh I was testing out the general version and forgot to revert back to :cdc_sample_layout